dfa/scratch_pred.py

```diff
@@ -0,0 +1,33 @@
+from pathlib import Path
+
+import torch
+
+from dfa.audio import Audio
+from dfa.model import Aligner
+from dfa.text import Tokenizer
+
+if __name__ == '__main__':
+
+    checkpoint = torch.load('/Users/cschaefe/dfa_checkpoints/latest_model.pt', map_location=torch.device('cpu'))
+    config = checkpoint['config']
+    symbols = checkpoint['symbols']
+    audio = Audio(**config['audio'])
+    tokenizer = Tokenizer(symbols)
+    model = Aligner.from_checkpoint(checkpoint).eval()
+    print(f'model step {model.get_step()}')
+
+    main_dir = Path('/Users/cschaefe/datasets/audio_data/Cutted_merged')
+    file_id = '04902'
+
```

Comment:
would be nice to already use the Paths object

Reply:
Absolutely,  thats just the sketch